platform/platform-filter-delegate/src/main/java/org/codice/ddf/platform/filter/delegate/DelegateServletFilter.java

```diff
@@ -288,7 +288,7 @@ public String getInitParameter(String name) {
                 ? ""
                 : securityFilterServiceReference.getProperty(key).toString();
 
-        final String HTTP_WHITEBOARD_SERVLET_INIT_PARAM_PREFIX = "servlet.init.";
+        final String HTTP_WHITEBOARD_SERVLET_INIT_PARAM_PREFIX = "servlet\\.init\\.";
```

Comment:
❗️  I believe we have an issue here. 

The escaping is good for the call to `replaceFirst()` since it expects a regex but not for `startsWith()` which doesn't. So I think the original code was ok since the test for `startsWith()` would have already checked that it was an actual dot (i.e. .) and not any characters.